cohortextractor/trino_utils.py

```diff
@@ -54,16 +55,35 @@ def adapt_connection(conn, conn_params, env=os.environ):
     """
 
     session = requests.Session()
-    crt = env.get("PRESTO_TLS_CERT")
-    key = env.get("PRESTO_TLS_KEY")
```

Comment:
This feels slightly overcomplicated to me as we're now duplicating various bits of code including writing temp files twice with different prefixes. I think you could have achieved the same thing with just:
```py
crt = env.get("TRINIO_TLS_CERT", default=env.get("PRESTO_TLS_CERT"))
key = env.get("TRINIO_TLS_KEY", default=env.get("PRESTO_TLS_KEY"))
```
Which would have been easier to back out.

But fine to leave as is, as ought to be short lived in any case.

Reply:
Here the intent is to guard against unexpected mixing and matching of different environment variables. For example, if we fallback to the `default` here, someone could supply **`TRINO`**`_TLS_CERT`, but specify **`PRESTO`**`_TLS_KEY`. This would work now. It would fail later when we only look for `TRINO_TLS_CERT` and `TRINO_TLS_KEY`.  This could catch "future me" (or "future someone else") out.

But, we're likely the only users actually specifying these variables. It may not matter too much to protect against this.

Agree that if it was to stay in the code perpetually, the duplication is more complex. But we should be able to revert back to the original structure that was there after removing the `PRESTO` variables.

On balance then, I'm happy to leave as is.

***

Relating to file prefixes, I'm not completely certain if the filenames are checked/loaded elsewhere. I don't think so from a quick search in `opensafely-core` for anything relevant.